### ISSUE TITLE
fill in the missing bits 

### ISSUE THREAD

**Issue by rsuchecki** [csiro-data-school/hpc#8]
;-)

**Comment by rsuchecki** [comment]
More seriously,  should we arrange a webex meeting once @OH-AU is back and spread the load a bit @madeleine-barton?

**Comment by spriggsy83** [comment]
Might be good to have a plan.  I figured I was just going to pick an episode per night and spend an hour adding to it, each night for the next couple of weeks.  Was going to start last night, but in too late.
I had questions though, seeing as @OH-AU has taught this class before, probably has some pre-existing material, and as an IM&T representative, might have some ideas of the "right" way of doing things that might not be the way I do them...
But still, my plan was just to start chipping away, a little per night, I hope.

**Comment by madeleine-barton** [comment]
I agree, a plan would be good! I'm unfortunately a bit out of action for the next two weeks (have an annual project meeting that's taking priority at the moment). From March onwards, I can also put aside an hour or so a day to chip away at the episodes. And given my relative inexperience with the hpc, happy to volunteer myself as a guinea pig for any exercises that might need testing. @spriggsy83 are you able to merge and commit changes to the repo?

**Comment by spriggsy83** [comment]
Yep.
